app/services/books/search_strategies/s1/strategy.rb

```diff
@@ -49,10 +49,11 @@ def search_body(query_string)
         query: {
           simple_query_string: {
             fields: %w(title contextTitle visible_content),
-            query: query_string,
-            flags: "PHRASE|WHITESPACE",
+            query: fuzzify(query_string),
```

Comment:
did you mention that maybe the stuff above at `query_string.gsub` is no longer necessary anymore?

Reply:
Erika wanted to try fuzzy search again as well, but this time with the fuzzy_prefix_length set to 3 so mistakes in the first 3 characters are ignored

Reply:
right but i meant for the quote replacement

Reply:
The gsub stuff above is part of the fuzzy search so it's necessary. The thing that can be removed is the smart quotes char filter for English specifically, since I added asciifolding to map all non-ascii characters to ascii for English (but not for Spanish/Polish since they use non-ascii characters)